tests/robustness/scenarios.go

```diff
@@ -198,5 +198,24 @@ func regressionScenarios(t *testing.T) []testScenario {
 			cluster:   *e2e.NewConfig(opts...),
 		})
 	}
+	scenarios = append(scenarios, testScenario{
+		name:      "Issue17780",
+		failpoint: failpoint.AllowBatchCompactBeforeSetFinishedCompactPanic,
+		profile: traffic.Profile{
```

Comment:
Let's wait for #17968 to be fixed and then remove custom traffic profile

Reply:
Updated